app/serializers/api/customer_serializer.rb

```diff
@@ -1,5 +1,6 @@
 module Api
   class CustomerSerializer < ActiveModel::Serializer
-    attributes :id, :enterprise_id, :name, :code, :email, :allow_charges
+    attributes :id, :enterprise_id, :name, :code, :email, :allow_charges,
+               :gateway_recurring_payment_client_secret, :gateway_shop_id
```

Comment:
Just wondering; does `:gateway_recurring_payment_client_secret` always need to be exposed here? Or to put it another way, should it be exposed in every context where `Api::CustomerSerializer` is used?

Reply:
It definitely does not. Good catch. I'll change it to only return the fields if they're not nil.

Reply:
0d0665d515c4482b9919132b904d3b6902e7f458 👍